src/app/lists/loading.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Loading: React.FC<{}> = () => {
+    return <h1> Loading </h1>;
```

Comment:
please delete the `lists/loading.tsx` :)

Reply:
Done